cli/commands.js

```diff
@@ -99,6 +99,7 @@ const defaultCommands = {
     --ignore-application  # Ignores application generation                                         Default: false
     --ignore-deployments  # Ignores deployments generation                                         Default: false
     --skip-ui-grouping    # Disable the UI grouping behavior for entity client side code           Default: false
+    --skip-db-changelog   # Disable the generator of database changelogs                           Default: false
```

Comment:
generation

Reply:
there's just this tiny wording, and it's good to go :)

Reply:
I've fixed the typo, thanks for the review :)